pages/linux/arecord.md

```diff
@@ -0,0 +1,11 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
+
+- Record a snippet in "CD" quality:
+
+`arecord -vv -f cd output.wav`
+
+- Pass a raw recording for further processing:
```

Comment:
I tried this command and this is like the 1st one but saves the audio in `mp3` format with the `| lame -r - output.mp3` the audio is output to stdout (terminal). I'm not sure about this example description, can another reviewer check?

Reply:
The effect should be saving the variable duration audio snippet as `output.mp3`. Doesn't happen for you?

Reply:
Improved the description in d1e19443434197a014503c05b061153485d7b1de

Reply:
Sorry, I had a typo I meant `without`.